packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/line.scope

```diff
@@ -0,0 +1,117 @@
+
+aaa
+
+---
```

Comment:
how do we handle a file with no trailing newline? I know we don't allow them in our source code, but we should be able to test our support for them in case users use them

Reply:
I have disabled pre commit on these files.

Reply:
yes but that only impacts the end of the file. I mean:

```
a
---
...
```

vs

```
a

---
...
```

I would think in the first case, the content is `a\n`, in the second the content is `a\n\n`, but then how do we represent just `a` (ie no trailing newline)?

Reply:
This is already `a`
```
a
---
```

Reply:
Huh. So a single line results in no trailing newline? Not sure how I feel about that. Files with no trailing newline are generally considered bad practice, so I'd lean towards the default being trailing newline cc/ @josharian

Reply:
Yes the block above the dashes is the entire document.